src/views/GovernanceReviewTeam/GRBReview/DocumentsTable.tsx

```diff
@@ -0,0 +1,332 @@
+
+        <p>
+          {t('technicalAssistance:documents.supportingDocuments.removeInfo')}
+        </p>
+
+        <Button
+          type="button"
+          onClick={() => {
+            deleteDocument({ variables: { id: fileToDelete.id } })
+              .then(() => {
+                showMessage(
+                  t(
+                    'technicalAssistance:documents.supportingDocuments.removeSuccess',
+                    {
+                      documentName: fileToDelete.fileName
+                    }
+                  ),
+                  {
+                    type: 'success'
+                  }
+                );
+              })
+              .catch(() => {
+                showMessage(
+                  t(
+                    'technicalAssistance:documents.supportingDocuments.removeFail'
+                  ),
+                  {
+                    type: 'error'
+                  }
+                );
+              });
+
+            setFileToDelete(null);
+          }}
+        >
+          {t(
+            'technicalAssistance:documents.supportingDocuments.removeDocument'
+          )}
+        </Button>
+
+        <Button
+          type="button"
+          className="margin-left-2"
+          unstyled
+          onClick={() => setFileToDelete(null)}
+        >
+          {t('technicalAssistance:documents.supportingDocuments.cancel')}
+        </Button>
+      </Modal>
+    );
+  };
+
+  return (
+    <div id="systemIntakeDocuments">
+      <ConfirmDeleteModal />
+
+      <Table bordered={false} fullWidth scrollable {...getTableProps()}>
+        <thead>
+          {headerGroups.map(headerGroup => (
+            <tr {...headerGroup.getHeaderGroupProps()}>
+              {headerGroup.headers.map(column => (
+                <th
+                  {...column.getHeaderProps(column.getSortByToggleProps())}
+                  aria-sort={getColumnSortStatus(column)}
+                  scope="col"
+                  className="border-bottom-2px"
+                >
+                  <Button
+                    type="button"
+                    unstyled
+                    className="width-full display-flex"
+                    {...column.getSortByToggleProps()}
+                  >
+                    <div className="flex-fill text-no-wrap">
+                      {column.render('Header')}
+                    </div>
+                    <div className="position-relative width-205 margin-left-05">
+                      {getHeaderSortIcon(column)}
+                    </div>
+                  </Button>
+                </th>
+              ))}
+            </tr>
+          ))}
+        </thead>
+        <tbody {...getTableBodyProps()}>
+          {page.map(row => {
+            prepareRow(row);
+            return (
+              <tr {...row.getRowProps()}>
+                {row.cells.map(cell => {
+                  return (
+                    <td {...cell.getCellProps()}>{cell.render('Cell')}</td>
```

Comment:
Table cells need `vertical-align: top` style added.

Screenshot from Figma:
![Screenshot 2024-08-26 at 12 00 51 PM](https://github.com/user-attachments/assets/7efaf07a-3e8d-491f-8672-74fa9de9d2db)

Reply:
`<td className='text-ttop' ... >` seems to work here, is that OK?

Reply:
Yep that works!

Reply:
Added in [f2c84ba](https://github.com/CMS-Enterprise/easi-app/pull/2776/commits/f2c84ba292c27c814f60340ec34482be643dd977)